build-system/tasks/extension-helpers.js

```diff
@@ -597,7 +598,24 @@ function buildExtensionCss(extDir, name, version, options) {
       })
     );
   }
-  promises.push(mainCssBinary);
+  promises.push(mainCssBinaryPromise);
+
+  // Currently JSON.stringifying to allow arrays and strings:
+  // {"wrapper": "bento"} and {"wrapper": ["bento"]}
+  // TODO(alanorozco): We might need a `bento` flag instead.
+  if (options.wrapper && JSON.stringify(options.wrapper).includes('"bento"')) {
+    // TODO(alanorozco): Use a TransformCache
```

Comment:
It's likely unnecessary to use a `TransformCache` since this is relatively fast. Opinions welcome.

On my machine:

```
[09:53:08] Transformed bento-accordion.css (089 ms)
[09:53:08] Transformed bento-base-carousel.css (079 ms)
[09:53:08] Transformed bento-facebook.css (073 ms)
[09:53:08] Transformed bento-embedly-card.css (070 ms)
[09:53:08] Transformed bento-fit-text.css (069 ms)
[09:53:08] Transformed bento-instagram.css (059 ms)
[09:53:08] Transformed bento-inline-gallery.css (062 ms)
[09:53:08] Transformed bento-lightbox.css (058 ms)
[09:53:08] Transformed bento-lightbox-gallery.css (054 ms)
[09:53:08] Transformed bento-selector.css (046 ms)
[09:53:08] Transformed bento-date-display.css (074 ms)
[09:53:08] Transformed bento-brightcove.css (079 ms)
[09:53:08] Transformed bento-date-countdown.css (076 ms)
[09:53:08] Transformed bento-timeago.css (011 ms)
[09:53:08] Transformed bento-stream-gallery.css (010 ms)
[09:53:08] Transformed bento-vimeo.css (005 ms)
[09:53:08] Transformed bento-twitter.css (007 ms)
[09:53:08] Transformed bento-video-iframe.css (003 ms)
[09:53:08] Transformed bento-video.css (005 ms)
[09:53:08] Transformed bento-youtube.css (005 ms)
[09:53:08] Transformed bento-social-share.css (042 ms)
[09:53:08] Transformed bento-soundcloud.css (040 ms)
[09:53:08] Transformed bento-sidebar.css (044 ms)
[09:53:08] Transformed bento-wordpress-embed.css (003 ms)
[09:53:08] Recompiled all CSS files into build/ (140 ms)
````

Reply:
I agree with your intuition -- caching can always be added later as well.